src/modules/Elsa.Workflows.Management/Models/TimestampFilter.cs

```diff
@@ -0,0 +1,26 @@
+using Elsa.Workflows.Management.Enums;
+using JetBrains.Annotations;
+
+namespace Elsa.Workflows.Management.Models;
+
+/// <summary>
+/// Represents a timestamp filter used for filtering data based on a specified timestamp column and operator.
+/// </summary>
+[UsedImplicitly]
+public class TimestampFilter
+{
+    /// <summary>
+    /// Gets or sets the column to filter by.
+    /// </summary>
+    public string Column { get; set; } = default!;
```

Comment:
```suggestion
    public required string Column { get; set; } = default!;
```

Reply:
required if for:
if TimestampFilter not null should have all properties not null

Reply:
I wasn't aware of this capability - very nice 👍🏻